### PR TITLE
Fix boxplots

### PR DESCRIPTION
**Pull request by rpkamp**
Based on https://github.com/SURFnet/learning-analytics/pull/61

Fixes https://github.com/SURFnet/learning-analytics/issues/58

TODO:
- [x] Review https://github.com/SURFnet/learning-analytics/pull/61 first

### CODE REVIEW

File: src/AppBundle/Manager/QuartileCalculator.php [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+        return [
+            'q0' => $array[0],
+            'q1' => self::median(array_slice($array, 0, $q1Index)),
+            'q2' => self::median($array),
+            'q3' => self::median(array_slice($array, $q3Index)),
+            'q4' => $array[$count - 1],
+        ];
+    }
+
+    private static function median($array)
```
Comment (seggen-ibuildings) [inline src/AppBundle/Manager/QuartileCalculator.php]: docblock and type hint?

File: tests/AppBundle/Manager/QuartileCalulcatorTest.php [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+                'expected' => ['q0' => 1, 'q1' => 1, 'q2' => 1, 'q3' => 1, 'q4' => 1],
+            ],
+            'odd number of elements' => [
+                'in' => [1, 2, 3, 4, 5],
+                'expected' => ['q0' => 1, 'q1' => 1.5, 'q2' => 3, 'q3' => 4.5, 'q4' => 5],
+            ],
+            'even number of elements' => [
+                'in' => [1, 2, 3, 4, 5, 6],
+                'expected' => ['q0' => 1, 'q1' => 2, 'q2' => 3.5, 'q3' => 5, 'q4' => 6],
+            ],
```
Comment (seggen-ibuildings) [inline tests/AppBundle/Manager/QuartileCalulcatorTest.php]: please add datasets for 7 and 8 elements as well

File: src/AppBundle/Manager/StatsManager.php [hunk 3]
```diff
@@ -174,14 +177,15 @@ public function getLabelCounts(Recipe $recipe, Course $course, Period $period, U
                 ];
             }
 
-            $data[$count['object']][$count['verb']][$count['label']]['stamps'][] = $count['timestamp']->format('U');
```
Comment (seggen-ibuildings) [inline src/AppBundle/Manager/StatsManager.php]: please remove `'stamps' => [],` above

File: tests/AppBundle/Manager/QuartileCalulcatorTest.php (hunk 2 above)
Comment (rpkamp) [inline tests/AppBundle/Manager/QuartileCalulcatorTest.php]: Done

File: src/AppBundle/Manager/StatsManager.php (hunk 3 above)
Comment (rpkamp) [inline src/AppBundle/Manager/StatsManager.php]: Done

**Pull request merged**